src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -121,7 +121,20 @@ The storage type of ``sigmoid`` output is always dense
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseOut{"_backward_sigmoid"});
 
 MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU(_backward_sigmoid,
-                                               unary_bwd<mshadow_op::sigmoid_grad>);
+                                               unary_bwd<mshadow_op::sigmoid_grad>)
+.set_attr<nnvm::FGradient>("FGradient",
+  [](const nnvm::NodePtr& n, const std::vector<nnvm::NodeEntry>& ograds) {
+    auto fx = nnvm::NodeEntry{n->inputs[1]};
+    auto gx_ograd = nnvm::NodeEntry{n};
+
+    std::vector<nnvm::NodeEntry> ret;
+       
+    ret.emplace_back(MakeNode("elemwise_mul", n->attrs.name + "_backward_grad_grad_inp",
```

Comment:
Shouldn't the first item be _backward_grad_grad and the second item be _backward_grad_grad_inp?

Given that:
```
n->input[0]:  ygrad
n->input[1]: y
```

Reply:
Actually the code is incorrect in terms of naming the attribute and probably variable names.
As my main concern was trying to figure why we `f(x) * f'(x)` where `f(x)` was expected.